transport-native-unix-common/src/main/c/netty_unix_socket.c

```diff
@@ -289,26 +320,10 @@ static jobject _recvFrom(JNIEnv* env, jint fd, void* buffer, jint pos, jint limi
     return createDatagramSocketAddress(env, &addr, res);
 }
 
-static void netty_unix_socket_optionHandleError(JNIEnv* env, int err, char* method) {
-    if (err == EBADF) {
```

Comment:
netty_unix_socket_setOption0 needs to be defined before _socket() uses it, and I wanted to keep most of these functions together.  

I can move the minimal number of functions to do so, or move _socket down.  Preference?